### ISSUE TITLE
Manager and Share buttons do not appear - reopened

### ISSUE THREAD

**Issue by sapientsnap** [Comfy-Org/ComfyUI-Manager#867]
Hello!

[note: I’m very experienced with Comfy and I scrubbed the issues for a fix before posting this one…]

I’m on a Mac M1 OSX 13.6.1 16GB VRAM and recently had to wipe my system of all Python instances (to solve conflicting path issues), then reinstalled Comfy and all dependencies properly in a virtual environment. 

All is working fine except the Manager. I’m having the issue where when I launch Comfy the manager and share buttons are not visible. 

To address this message I’ve reinstalled Manager twice using two different methods, the second time I used  Installation [method4] (Installation for linux+venv: ComfyUI + ComfyUI-Manager). 

Help from my ComfyCommunity friends is greatly appreciated!!  Here is the full startup log followed by an error in the ComfyUI log:

(venvSD) MoAI@Morales-M1-16 ComfyUI % python main.py
[START] Security scan
[DONE] Security scan

ComfyUI-Manager: installing dependencies done.

** ComfyUI startup time: 2024-07-14 20:26:11.804393
** Platform: Darwin
** Python version: 3.12.4 (v3.12.4:8e8a4baf65, Jun 6 2024, 17:33:18) [Clang 13.0.0 (clang-1300.0.29.30)]
** Python executable: /Users/MoAI/AI/StableDiffusion/venvSD/bin/python3
** ComfyUI Path: /Users/MoAI/AI/StableDiffusion/ComfyUI
** Log path: /Users/MoAI/AI/StableDiffusion/ComfyUI/comfyui.log

Prestartup times for custom nodes:
0.3 seconds: /Users/MoAI/AI/StableDiffusion/ComfyUI/custom_nodes/ComfyUI-Manager

Total VRAM 16384 MB, total RAM 16384 MB
pytorch version: 2.3.1
Set vram state to: SHARED
Device: mps
Using sub quadratic optimization for cross attention, if you have memory or speed issues try using: --use-split-cross-attention

Loading: ComfyUI-Manager (V2.46.3)

ComfyUI Revision: 2362 [a3dffc44] | Released on '2024-07-13'

Import times for custom nodes:
0.0 seconds: /Users/MoAI/AI/StableDiffusion/ComfyUI/custom_nodes/websocket_image_save.py
0.1 seconds: /Users/MoAI/AI/StableDiffusion/ComfyUI/custom_nodes/ComfyUI-Manager

Starting server

To see the GUI go to: http://127.0.0.1:8188/
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json

Here is an error listed Comfy’s log:

console
error
20:40:18.516Z
[
"Error loading extension",
"/extensions/ComfyUI-Manager/components-manager.js",
"SyntaxError\nImporting module '../../scripts/ui/draggableList.js' is not found.\n"
]

**Comment by ltdrdata** [comment]
Turn off the browser logger setting.
![image](https://github.com/user-attachments/assets/ae71f27e-78d6-4d6d-ba41-bfe09e2a8561)

And show me the stacktrace in browser console like this.
![image](https://github.com/user-attachments/assets/99abba99-03d1-4948-b325-978e92e23a69)

**Comment by sapientsnap** [comment]
Hi @ltdrdata, Thank you for responding. Late last night, for the fourth time, I wiped the venv completely, and reinstalled using a procedure I followed from YouTube some months ago. To my disappointment, nothing changed. 

Out of frustration, I gave up for the night and just had fun playing around generating images. Since I had earlier (accidentally) wiped ALL my checkpoint models and LoRAs (~200Gb worth!) I started rebuilding my inventory of models. After about an hour of this, the manager and share buttons just appeared. I did nothing but just used the app.

The only thing I can think of is that the browser refreshed and overwrote the cache of the earlier non-working code?? Can’t say for sure since over this frustrating journey, I had used several different browsers and “private mode” in case browser cache was causing old code to override the reinstalls.  

So, good news: it’s all working, bad news, I don’t know why or what changed or how to fix it in the future if it happens again. 

Thank you for jumping in to assist!

**Comment by sapientsnap** [comment]
Update:

I have clean-installed Comfy and Manager several times using several different methods to no affect. 

What I learned is that mine is a browser cache problem. Out of extreme frustration the other day, I refreshed my browser like a machine gun, hitting the refresh about a dozen times in rapid succession and guess what happened?… The manager appeared. 

😳

When IU refreshed again, it disappeared. 😡

Now, I’ve learned that when I launch, the manager never shows up, but if I refresh 5~50 times, it will appear.  Very weird.  This is on both DuckDuckGo and Safari. Today I just simply cleared the browser cache and it launched properly the first time.

I wish I knew a few days ago it was a browser problem before spending countless hours troubleshooting in a different direction. To my credit, I did consider the browser issue early on, but only refreshing once or twice masked the issue which is resolved only by clearing the cache or reloading the page a billion times. 🙄
